src/App.php

```diff
@@ -130,7 +132,7 @@ public function run($request = null, $response = null)
         $httpFlow = $this->getEventManager();
 
         try {
-            $routerInfo = $dispatcher->dispatch($request);
+            $routerInfo = $dispatcher($request);
```

Comment:
@samsonasik the problem exists for php until 5.6.13 ([see the 3v4l link](https://3v4l.org/Rd6qm))